src/util/parameterize.ts

```diff
@@ -31,4 +34,13 @@ const parameterize = (obj: any, prefix?: string): string => {
   return str.join("&")
 }
 
+// IE does not encode by default like other browsers
+const maybeEncode = (value: string): string => {
+  const isBrowser = typeof window !== "undefined"
```

Comment:
This is already available as an export from [`util/env.ts`](https://github.com/graphiti-api/spraypaint.js/blob/master/src/util/env.ts#L1). Recommend we put `isIE` there too, and then this looks good.

Reply:
Will revisit when we figure out the best way to mock